Assets/Scripts/enemyAI.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class enemyAI : MonoBehaviour
+{
+	public float rotSpeed = 90f;
+
+	Transform player;
```

Comment:
You could make player a public or System.Serializable GameObject then put the player GameObject in the variable in the inspector. This would make sure that you always had the correct game object once you have the Player Game object you can use it's transform whenever you want!